docs/var.tfvars-doc.md

```diff
@@ -0,0 +1,203 @@
+# How to use var.tfvars
+
+- [How to use var.tfvars](#how-to-use-vartfvars)
+  - [Introduction](#introduction)
+    - [PowerVC Details](#powervc-details)
+    - [OpenShift Cluster Details](#openshift-cluster-details)
+    - [OpenShift Installation Details](#openshift-installation-details)
+    - [Misc Customizations](#misc-customizations)
+
+
+## Introduction
+
+This guide gives an overview of the various terraform variables that are used for the deployment.
+The default values are set in [variables.tf](../variables.tf)
+
+### PowerVC Details
+
+These set of variables specify the PowerVC details.
+
+```
+auth_url                    = "<https://<HOSTNAME>:5000/v3/>"
+user_name                   = "<powervc-login-user-name>"
+password                    = "<powervc-login-user-password>"
+tenant_name                 = "<tenant_name>"
+domain_name                 = "Default"
```

Comment:
Default -> <domain_name>

Reply:
Does PowerVC support any different domain name today ?

Reply:
I am not sure... @sudeeshjohn ?

Reply:
From this doc "Default" is the only supported value - https://www.ibm.com/support/knowledgecenter/SSVSPA_1.4.4/com.ibm.powervc.cloud.help.doc/powervc_pg_authenticatingwithpowervc_cloud.html